packages/compass-settings/src/components/sidebar.tsx

```diff
@@ -54,7 +53,8 @@ const SettingsSideNav: React.FunctionComponent<SidebarProps> = ({
       aria-labelledby="modal-title"
     >
       {items.map((item) => (
-        <Button
+        <button
```

Comment:
I don't think the settings navigation should have been leafygreen buttons anyway, because they just don't look like them. Same thing in the main / connetions sidebar. Menu items are just too custom.

Reply:
Small thing, can we update the commend on line 26 since it's no longer LeafyGreen? https://github.com/mongodb-js/compass/pull/3946/files#diff-e8dc5dba7d636d3aa573305a44ec9b4fafc28e1977a7a16e34e62afddc44ee5aR26